src/az390.java

```diff
@@ -10910,7 +10924,7 @@ private void fp_get_hex(){
 	try { // RPI 424
 		fp_big_dec1 = new BigDecimal(fp_text,fp_context);
 	} catch (Exception e){
-		log_error(162,"invalid decimal floating point constant");
+		log_error(162,"invalid decimal floating point constant" + fp_text); // #233 show text
```

Comment:
After "constant" please insert ": " (i.e. a colon and a space) for improved readability.

Reply:
ok, I agree.